caesars-palace/blocks/header/header.css

```diff
@@ -18,6 +18,23 @@ header .global-nav-desktop {
   display: none;
 }
 
+.dropdown-menu {
```

Comment:
Could you add a bit more specificity to your selectors? At least add `nav` but maybe even be more specific to `header nav`.